class pbconfig():
    pbconfig={'gds':{'requestClass':'PBMsgReqGds','requestProto':'PBMsgReqGds_pb2', 'responseClass':'PBMsgGDS','responseProto':'PBMsgGDS_pb2'}, 
                'assignHero':{'requestClass':'PBMsgReqAssignHero', 'requestProto':'PBMsgReqAssignHero_pb2', 'responseClass':'PBMsgAssignHero','responseProto':'PBMsgAssignHero_pb2'},
                'boostHero':{'requestClass':'PBMsgReqBoostHero','requestProto':'PBMsgReqBoostHero_pb2', 'responseClass':'PBMsgBoostHero','responseProto':'PBMsgBoostHero_pb2'},
                #'heroHouseList':{'requestClass':'PBMsgReqHeroHouseList','requestProto':'PBMsgReqHeroHouseList_pb2', 'responseClass':'PBMsgHeroHouseList','responseProto':'PBMsgHeroHouseList_pb2' }, 
               'heroItemList':{'requestClass':'PBMsgReqHeroItemList','requestProto':'PBMsgReqHeroItemList_pb2', 'responseClass':'PBMsgHeroItemList','responseProto':'PBMsgHeroItemList_pb2'}, 
               'heroExploreInit':{'requestClass':'PBMsgReqInitHeroExplore', 'requestProto':'PBMsgReqInitHeroExplore_pb2', 'responseClass':'PBMsgInitHeroExplore','responseProto':'PBMsgReqInitHeroExplore_pb2'}, 
               'heroExploreProcess':{'requestClass':'PBMsgReqProcessHeroExplore', 'requestProto':'PBMsgReqProcessHeroExplore_pb2', 'responseClass':'PBMsgProcessHeroExplore','responseProto':'PBMsgProcessHeroExplore_pb2'}, 
               'pve':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'','responseProto':''}, 
               'pve1':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'PBMsgPveInfo','responseProto':'PBMsgPveInfo_pb2'}, 
               'pve2':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'PBMsgPveStaminaInfo','responseProto':'PBMsgPveStaminaInfo_pb2'}, 
               'pve3':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'PBMsgPveStaminaInfo','responseProto':'PBMsgPveStaminaInfo_pb2'}, 
               'pve4':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'PBMsgPveMarchInfo','responseProto':'PBMsgPveMarchInfo_pb2'},
               'pve5':{'requestClass':'PBMsgReqPve', 'requestProto':'PBMsgReqPve_pb2', 'responseClass':'PBMsgPveResultInfo','responseProto':'PBMsgPveResultInfo_pb2'}, 
               'unassignHero':{'requestClass':'PBMsgReqUnassignHero', 'requestProto':'PBMsgReqUnassignHero_pb2', 'responseClass':'PBMsgUnassignHero','responseProto':'PBMsgUnassignHero_pb2'},
               'getMysteryChestList_proto':{'requestClass':'MystryChestRequest', 'requestProto':'PBMsgMystryChest_pb2', 'responseClass':'MystryChestResponse','responseProto':'PBMsgMystryChest_pb2'}, 

'heroHouseList':{'requestClass':'PBMsgReqHeroHouseList','requestProto':'PBMsgReqHeroHouseList_pb2','responseClass':'PBMsgHeroHouseList','responseProto':'PBMsgHeroHouseList_pb2'},
}
